test/parallel_api/numeric/numeric.ops/inclusive_scan_by_segment.pass.cpp

```diff
@@ -53,31 +128,94 @@ struct test_inclusive_scan_by_segment
         }
     }
 
-    template <typename Iterator1, typename Iterator2, typename Size>
+#ifdef DUMP_CHECK_RESULTS
+    template <typename Iterator, typename Size>
+    void display_param(const char* msg, Iterator it, Size n)
+    {
+        using DifferenceType = typename DifferenceTypeT<Iterator>::DifferenceType;
+
+        std::cout << msg;
+        for (DifferenceType i = 0; i < n; ++i)
+        {
+            if (i > 0)
+                std::cout << ", ";
+            std::cout << it[i];
+        }
+        std::cout << std::endl;
+    }
+#endif // DUMP_CHECK_RESULTS
+
+    template <typename Iterator1, typename Iterator2, typename Iterator3, typename Size>
     void
-    check_values(Iterator1 host_keys, Iterator2 val_res, Size n)
+    check_values(Iterator1 host_keys, Iterator2 host_vals, Iterator3 val_res, Size n)
```

Comment:
I believe that one `check_values` written below on line 159 enough.

Reply:
Fixed

Reply:
No, it's not enough.

Reply:
Fixed